src/main/kotlin/no/nav/familie/ba/sak/kjerne/vedtak/vedtaksperiode/brevBegrunnelseProdusent/BrevBegrunnelseProdusent.kt

```diff
@@ -154,6 +155,12 @@ private fun hentStandardBegrunnelser(
         it.erGjeldendeForReduksjonFraForrigeBehandling(begrunnelseGrunnlag)
     }
 
+    val filtrertPåOpphørFraForrigeBehandling = filtrertPåRolle.filterValues {
+        vedtaksperiode.type == Vedtaksperiodetype.OPPHØR && it.erGjeldendeForOpphørFraForrigeBehandling(
```

Comment:
`vedtaksperiode.type == Vedtaksperiodetype.OPPHØR` tester vel det samme som `begrunnelseGrunnlag !is BegrunnelseGrunnlagForPeriodeMedOpphør`. Kan vi bare fjerne den?

![image](https://github.com/navikt/familie-ba-sak/assets/17828446/428c38f7-e533-442e-8b98-4d365ee315f7)

```suggestion
        it.erGjeldendeForOpphørFraForrigeBehandling(
```

Reply:
Jepp - la inn denne sjekken før jeg lagde eget begrunnelsesgrunnlag